app/components/files/file_icon.tsx

```diff
@@ -49,7 +50,7 @@ const styles = StyleSheet.create({
 
 const FileIcon = ({
     backgroundColor, defaultImage = false, failed = false, file,
-    iconColor, iconSize = 48, smallImage = false,
+    iconColor, iconSize = 48, smallImage = false, testID = 'file-icon',
```

Comment:
This change has nothing to do with the actual fix.  FileIcon is being used inside bookmark_icon and I needed a testID.  Could always mock this instead of making changes.  But adding testID is benign, so I don't see any harm in this.